lib/matplotlib/axes/_axes.py

```diff
@@ -876,6 +876,55 @@ def _to_points(xy1, xy2, slope):
         self._request_autoscale_view()
         return line
 
+    def axline_transaxes(self, xy1, slope, **kwargs):
+        """
+        Add an infinitely long straight line.
+
+        The line is defined by one point in axes coordinates *xy1* and a
+        *slope* in data coordinates.
+
+        This should only be used with linear scales; the *slope* has no clear
+        meaning for all other scales, and thus the behavior is undefined.
+
+        Parameters
+        ----------
+        xy1 : (float, float)
+            Point for the line to pass through, in axes coordinates.
```

Comment:
```suggestion
            Point for the line to pass through, in axes coordinates.  (0,0) is bottom left corner of the axes, (1, 1) is upper right. 
```

Reply:
added in 527aa460d25dfbd43229bdae6308b162db9eb902.